docs/reference.rst

```diff
@@ -31,6 +31,18 @@ history:
     >>> Poll.history.count()
     1000
 
+If you want to specify a change reason for each record in the bulk create, you
+can add `changeReason` on each instance:
+
+.. code-block:: pycon
+
+    >>> for poll in data:
+            poll.changeReason = 'reason'
+    >>> objs = bulk_create_with_history(data, Poll, batch_size=500)
+    >>> Poll.objects.get(id=data[0].id).history_change_reason
```

Comment:
Wouldn't this have to be `HistoricalPoll` or `Poll.history.get`? I don't think there's a `history_change_reason` on the base instance